thirdparty/misc/pcg.cpp

```diff
@@ -13,3 +13,14 @@ uint32_t pcg32_random_r(pcg32_random_t* rng)
     uint32_t rot = oldstate >> 59u;
     return (xorshifted >> rot) | (xorshifted << ((-rot) & 31));
 }
+
+// -- GODOT start --
```

Comment:
I think there's no need to add the `// -- GODOT start --` comments here as it's code from the same upstream. Just add a link to its source so that we understand where these two methods were taken from.

Reply:
Yeah, I think the same. Where to add this link ? Like so ? 
![image](https://user-images.githubusercontent.com/3036176/55093921-67fea880-50c6-11e9-8971-7e60db4bc97c.png)

Reply:
Ok, changed it like the picture above